frontend/src/services/ApplicationState.ts

```diff
@@ -5,7 +5,7 @@ import { computed, ref } from 'vue';
 export const applicationsUserAdministers = ref<FamApplication[]>([]);
 
 // The application selected by the user to admin
-export const selectedApplication = ref<FamApplication>();
+export const selectedApplication = ref<FamApplication | null>();
```

Comment:
let's change "null" to "undefined", so it won't complain the type error when we use it

Reply:
Usually undefined is when a variable is declared but a value has not been assigned. Do you prefer to use it instead of null for this case?

Reply:
yeah... maybe we don't need to change this, I saw a type complain before, but I just checked again, it disappeared... maybe I just need a refresh